tests/avifgridapitest.c

```diff
@@ -29,7 +29,7 @@ static avifBool createImage(int width, int height, int depth, avifPixelFormat yu
     uint32_t uvWidth = ((*image)->width + formatInfo.chromaShiftX) >> formatInfo.chromaShiftX;
     uint32_t uvHeight = ((*image)->height + formatInfo.chromaShiftY) >> formatInfo.chromaShiftY;
 
-    for (uint32_t plane = 0; plane < (formatInfo.monochrome ? 1 : AVIF_PLANE_COUNT_YUV); ++plane) {
+    for (uint32_t plane = 0; plane < (formatInfo.monochrome ? 1U : AVIF_PLANE_COUNT_YUV); ++plane) {
```

Comment:
Since `AVIF_PLANE_COUNT_YUV` is defined as 3, not 3U:

`#define AVIF_PLANE_COUNT_YUV 3`

won't `AVIF_PLANE_COUNT_YUV` also cause the signed/unsigned mismatch warning?

It seems simpler to just declare `plane` as `int` (and also use a local variable as you suggested in Chat):

```
const int planeCount = formatInfo.monochrome ? 1 : AVIF_PLANE_COUNT_YUV;
for (int plane = 0; plane < planeCount; ++plane) {
```

Reply:
I agree. I've updated the commit to unify the look of all y4m-related plane iterations. They all now use the same variable names and types.